WordPress/Classes/ViewRelated/Views/List/NotificationsList/NotificationsTableViewCellContent.swift

```diff
@@ -0,0 +1,206 @@
+            self.info = info
+        }
+
+        var body: some View {
+            HStack(alignment: .top, spacing: 0) {
+                avatarHStack
+                textsVStack
+                    .offset(x: -info.avatarStyle.leadingOffset*2)
+                    .padding(.horizontal, Length.Padding.split)
+                if let actionIconName = info.actionIconName {
+                    actionIcon(withName: actionIconName)
+                }
+            }
+            .padding(.trailing, Length.Padding.double)
+        }
+
+        private var avatarHStack: some View {
+            HStack(spacing: 0) {
+                if info.shouldShowIndicator {
+                    indicator
+                        .padding(.horizontal, Length.Padding.single)
+                    AvatarsView(style: info.avatarStyle)
+                        .offset(x: -info.avatarStyle.leadingOffset)
+                } else {
+                    AvatarsView(style: info.avatarStyle)
+                        .offset(x: -info.avatarStyle.leadingOffset)
+                        .padding(.leading, Length.Padding.medium)
+                }
+            }
+        }
+
+        private var indicator: some View {
+            Circle()
+                .fill(Color.DS.Background.brand(isJetpack: AppConfiguration.isJetpack))
+                .frame(width: Length.Padding.single)
+        }
+
+        private var textsVStack: some View {
+            VStack(alignment: .leading, spacing: 2) {
+                Text(info.title)
+                    .style(.bodySmall(.regular))
+                    .foregroundStyle(Color.DS.Foreground.primary)
+                    .lineLimit(2)
+
+                if let description = info.description {
+                    Text(description)
+                        .style(.bodySmall(.regular))
+                        .foregroundStyle(Color.DS.Foreground.secondary)
+                        .lineLimit(2)
+                }
+            }
+        }
+
+        private func actionIcon(withName iconName: String) -> some View {
+            Image(systemName: iconName)
+                .imageScale(.small)
+                .foregroundStyle(Color.DS.Foreground.secondary)
+                .frame(width: Length.Padding.medium, height: Length.Padding.medium)
+        }
+    }
+}
+
+// MARK: - Regular Style
+fileprivate extension NotificationsTableViewCell.Content {
+    struct Altered: View {
+        private let info: Style.Altered
+
+        fileprivate init(info: Style.Altered) {
+            self.info = info
+        }
+
+        var body: some View {
+            // To not pollute the init too much, colors are uncustomizable
+            // If a need arises, they can be added to the `Altered.Info` struct.
+            HStack(spacing: 0) {
+                Group {
+                    Text(info.text)
+                        .style(.bodySmall(.regular))
+                        .foregroundStyle(Color.white)
+                        .lineLimit(1)
```

Comment:
[Suggestion] This text is cut off when fonts are enlarged. I think increasing the line limit to `2` should fix it.

![CleanShot 2024-02-09 at 13 53 51@2x](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/61d7a82f-cc66-4bff-a2f9-44c242974745)